iquip/apps/scheduler.py

```diff
@@ -0,0 +1,352 @@
+        self.queueView.setItemDelegate(ExperimentDelegate(self.queueView))
+        self.queueView.setMovement(QListView.Free)
+        self.queueView.setObjectName("Queued Experiments")
+        self.model = ExperimentModel()
+        self.queueView.setModel(self.model)
+        # layout
+        layout = QVBoxLayout(self)
+        layout.setContentsMargins(0, 0, 0, 20)
+        layout.addWidget(QLabel("Currently running:"))
+        layout.addWidget(self.runningView)
+        layout.addWidget(QLabel("Queued experiments:"))
+        layout.addWidget(self.queueView)
+
+
+class RunningExperimentView(QWidget):
+    """Widget for displaying the information of the experiment, especially for the one running.
+    
+    Attributes:
+        experimentInfo: The ExperimentInfo instance that holds the experiment information.
+        argsLayout: The HBoxLayout for displaying the experiment information besides its name.
+        nameLabel: The QLabel instance for displaying the experiment name.
+    """
+
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent=parent)
+        self.experimentInfo = None
+        # widgets
+        self.nameLabel = QLabel("None", self)
+        # layout
+        layout = QVBoxLayout(self)
+        self.argsLayout = QHBoxLayout()
+        layout.addWidget(self.nameLabel)
+        layout.addLayout(self.argsLayout)
+
+    def updateInfo(self, info: Optional[ExperimentInfo] = None):
+        """Updates the information by modification.
+        
+        This updates SchedulerFrame.runningView when new experiment starts to run.
+
+        Args:
+            info: The experiment information. None if there is no experiements running.
+        """
+        _dismiss_items(self.argsLayout)
+        self.experimentInfo = info
+        if info is not None:
+            self.nameLabel.setText(info.name)
+            for key, value in info.arginfo.items():
+                if key != "priority":
+                    self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
+        else:
+            self.nameLabel.setText("None")
+
+
+class ExperimentView(QWidget):
+    """Widget for displaying the information of the experiment.
+    
+    Attributes:
+        experimentInfo: The ExperimentInfo instance that holds the experiment information.
+        layout: The list of ExperimentView widget.
+        args: The list of QLabel widget displaying the experiment information.
+        argsLayout: The HBoxLayout for displaying the experiment information besides its name.
+        nameLabel: The QLabel instance for displaying the experiment name.
+        editButton: The button to call frame for edition of the experiment.
+    """
+
+    def __init__(self, info: ExperimentInfo, parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            info: The information of the experiment.
+            parent: The widget that contains this widget.
```

Comment:
Similar to the review in the constructor of `SchedulerApp`, you should remove the `parent` docstring!